src/build/leo/aleo-sdk/js-ts/guide.md

```diff
@@ -1 +1,1187 @@
+
+The SDK provides the ability to execute Aleo Instructions programs %100 client-side within the browser.
+
+The `ProgramManager` object encapsulates the functionality for executing programs and making zero knowledge proofs about
+them. Under the hood it uses cryptographic code compiled from [snarkVM](https://developer.aleo.org/aleo) into WebAssembly.
+JavaScript bindings to this WebAssembly code allows execution of programs in zero knowledge fully within the browser
+without requiring any external communication with the internet. Users interested in lower level details on how this is
+achieved can visit the [aleo-wasm](https://github.com/AleoHQ/sdk/tree/testnet3/wasm) crate.
+
+The basic execution flow of a program is as follows:
+1. A web app is loaded with an instance of the `ProgramManager` object
+2. An Aleo program in `Aleo Instructions` format is loaded into the `ProgramManager` as a wasm object
+3. The web app provides a user input form for the program
+4. The user submits the inputs and the zero knowledge execution is performed client-side within WebAssembly
+5. The result is returned to the user
+6. (Optional) A fully encrypted zero knowledge transcript of the execution is optionally sent to the Aleo network
+
+A diagrammatic representation of the program execution flow is shown below.
+```mermaid
+graph LR
+    p1[Leo Program]
+    p2[Aleo Instructions]
+ 
+    subgraph Browser Web-App
+        subgraph ProgramManager
+            subgraph Aleo-Wasm-Module
+                in-memory-program
+            end
+        end
+    end
+
+    p1-->p2--load program--oin-memory-program-.ZK result.->user
+    user-.user input.->in-memory-program
+    in-memory-program-."ZK result (Optional)".->Aleo-Network
+```
+
+### 2.3 WebAssembly Initialization
+
+❗WebAssembly must be initialized before calling any SDK functions. The current Aleo SDK manages the wasm initialization. Therefore, the workers must be defined properly.
+
+Aleo programs are made zero knowledge through the usage of `ZkSnarks`. The Rust code behind Aleo programs and the ZkSnarks
+that make them zero knowledge are hosted in the [snarkVM Repository](https://github.com/AleoHQ/SnarkVM). The Aleo SDK
+compiles this code to WebAssembly and creates JavaScript bindings, enabling Aleo programs to run directly in the browser.
+
+Before any logic within the SDK is run within the browser however, the WebAssembly module the SDK contains must be
+initialized before any SDK functions can be executed. This is done simply by calling the `initializeWasm` function at a
+point in your code before any other SDK functions are called:
+```typescript
+import { Account, initializeWasm } from '@aleohq/sdk';
+
+// Assuming top-level await is enabled. This can also be initialized within a promise.
+await initializeWasm();
+
+/// Create a new Aleo account
+const account = new Account();
+````
+
+An example of how to initialize WebAssembly in a React app is shown in [Section 2.8](#28-React-Example)
```

Comment:
This link seems good but didn't work for me when clicking on it, i.e Section 2.8

Reply:
I tested it after your comment, but it seems to be working fine on my end. Maybe we can ask someone else to confirm that as well

Reply:
@ghostant-1017 Can you please check the reference?